RELEASE.md

```diff
@@ -0,0 +1,16 @@
+Release type: patch
+
+Added pytest-xdist, for faster tests.
+
+Before:
+```shell
+poetry run pytest
+
+=============== 1637 passed, 23 skipped, 20 xfailed, 37 xpassed, 27 warnings in 115.97s (0:01:55)  ===============
+```
+After:
+```shell
+poetry run pytest -n auto
+
+=============== 1637 passed, 23 skipped, 17 xfailed, 40 xpassed, 27 warnings in 53.34s ===============
+```
```

Comment:
I don't think we need a release for this 😊

Shall we use `-n auto` on github actions too?

Reply:
I want though I'll see if it passes.

Reply:
As I thought coverage can't figure out --apend...